### DISCUSSION TITLE
Possible to merge old TARDIS_World_name into the single Time Vortex world?

### DISCUSSION THREAD

**Discussion by Chew** [eccentricdevotion/TARDIS#446] [Q&A] [answered]
Hello, someone was wondering if it was possible to merge multiple Tardis worlds into the one vortex world. Definitely better on resources! 
If not, how would this be implemented? Would it need to grow in the new world?

Thanks!

**Comment by eccentricdevotion** [comment, marked as answer]
Not possible to merge worlds with Spigot API as such.
I presume they'd want TARDIS interiors - even if customised - to remain the same with all rooms etc...
Would need to copy blocks from one world as a schematic of some sort and paste them into the TimeVortex world in the appropriate TIPS slot - not something the plugin can currently do.
